be/src/exec/pipeline/analysis/analytic_sink_operator.cpp

```diff
@@ -90,10 +96,10 @@ Status AnalyticSinkOperator::push_chunk(RuntimeState* state, const vectorized::C
 
     _analytor->input_chunks().emplace_back(chunk);
 
-    return _process_by_partition_if_necessary();
+    return (this->*_process_by_partition_if_necessary)();
```

Comment:
```suggestion
    return this->_process_by_partition_if_necessary();
```

Reply:
Called object type 'starrocks::Status (starrocks::pipeline::AnalyticSinkOperator::*)()' is not a function or function pointer

Reply:
https://stackoverflow.com/questions/30886420/function-pointer-is-not-a-function-or-function-pointer